ProjectManager/ProjectManager/Scene/Main/ViewModel/MainViewModel.swift

```diff
@@ -42,7 +47,34 @@ final class MainViewModel: MainViewModelEvent, MainViewModelState, ErrorObservab
     }
     
     func viewDidLoad() {
-        fetchData()
+        startMonitoring()
+    }
+    
+    private func startMonitoring() {
+        let monitoringQueue = DispatchQueue(label: "network", attributes: .concurrent)
+        monitor.start(queue: monitoringQueue)
+        monitor.pathUpdateHandler = { [weak self] path in
+            if path.status == .satisfied {
+                self?.online.accept(true)
+                self?.syncronize()
+            } else {
+                self?.online.accept(false)
+                self?.fetchData()
+            }
+        }
```

Comment:
```suggestion
        monitor.pathUpdateHandler = { [weak self] path in
            if path.status == .satisfied {
                self?.online.accept(true)
                self?.syncronize()
            } else {
                self?.online.accept(false)
            }
            self?.fetchData()
        }
```
기존에는 `synchronize`에서 데이터 싱크를 맞추고 `fetchData`까지 해줬는데요.
`fetchData`가 `synchronize`의 역할은 아닌거 같습니다.
네트워크 온/오프라인 모두 `fetchData`를 사용하므로 위와 같이 변경할 수 있을거 같습니다.